src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj

```diff
@@ -4,7 +4,8 @@
 
   <PropertyGroup>
     <Description>NuGet's understanding of target frameworks.</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary);net40</TargetFrameworks>
```

Comment:
Use `IsBuildOnlyXPLATProjects` or just make it consistent throughout. 


Honestly, I'd be in favor of removing net40 sooner rather than alter.

Reply:
Made consistent with `IsBuildOnlyXPLATProjects`

I will remove net40 in another PR https://github.com/NuGet/NuGet.Client/pull/3513